src/autopas/containers/linkedCells/traversals/LCC08CellHandler.h

```diff
@@ -149,15 +152,29 @@ inline void LCC08CellHandler<ParticleCell, PairwiseFunctor, dataLayout, useNewto
     for (unsigned long y = 0ul; y <= _overlap[1]; ++y) {
       for (unsigned long z = 0ul; z <= _overlap[2]; ++z) {
         const unsigned long offset = cellOffsets[ov1_squared * x + ov1 * y];
+        const std::array<double, 3> offsetVec =
+            utils::ArrayUtils::static_cast_copy_array<double>(
+                utils::ThreeDimensionalMapping::oneToThreeD(offset, cellsPerDimension)) *
+            this->_cellLength;
         // origin
         {
-          // check whether cell is within interaction length
+          // check whether cell is within interaction length. distVec is the direction between borders of cells.
           const auto distVec =
               std::array<double, 3>{std::max(zero, x - one), std::max(zero, y - one), std::max(zero, z - one)} *
               _cellLength;
           const auto distSquare = utils::ArrayMath::dot(distVec, distVec);
           if (distSquare <= interactionLengthSquare) {
-            _cellPairOffsets.push_back(std::make_tuple(cellOffsets[z], offset, utils::ArrayMath::normalize(distVec)));
+            std::array<double, 3> baseCellVec = utils::ArrayUtils::static_cast_copy_array<double>(
+                utils::ThreeDimensionalMapping::oneToThreeD(cellOffsets[z], cellsPerDimension));
+
+            // Calculate the sorting direction from the base cell to the other cell.
+            std::array<double, 3> sortingDir = offsetVec - baseCellVec;
```

Comment:
Can just use `auto` for these cases.

Reply:
done in [8081014](https://github.com/AutoPas/AutoPas/pull/619/commits/80810142d34cf7872346976089a90e1753180bb7)